src/framework/components/anim/component.js

```diff
@@ -105,6 +105,7 @@ class AnimComponent extends Component {
             var layer = this.data.layers[i];
             for (var j = 0; j < layer.states.length; j++) {
                 var stateName = layer.states[j];
+                if (ANIM_CONTROL_STATES.includes(stateName)) continue;

```

Comment:
Not sure what happened to my previous comment, but `Array#includes` doesn't exist in IE11. Have you checked the Babel output to see if it's IE11-compatible?